examples/java/uk/co/real_logic/sbe/examples/ExampleUsingGeneratedStub.java

```diff
@@ -119,7 +119,7 @@ public static int encode(final Car car, final UnsafeBuffer directBuffer, final i
            .modelYear(2013)
            .available(BooleanType.TRUE)
            .code(Model.A)
-           .putVehicleCode(VEHICLE_CODE, srcOffset);
```

Comment:
That makes sense. I can remove this from the example. Does the generator code for these convenience method look ok?

Thank you!